kinode/packages/notify/pkg/manifest.json

```diff
@@ -0,0 +1,16 @@
+[
+    {
+        "process_name": "notify",
+        "process_wasm_path": "/notify.wasm",
+        "on_exit": "Restart",
+        "request_networking": false,
+        "request_capabilities": [
+            "http_server:distro:sys",
+            "homepage:homepage:sys"
+        ],
+        "grant_capabilities": [
+            "http_server:distro:sys"
+        ],
+        "public": true
```

Comment:
should the notify app really be public? shouldn't other apps need to request messaging capabilities in order to push notifications? wondering what the overall security model for this program should look like